### PR TITLE
66973 migrate docs yaml dap

### PR DESCRIPTION
**Pull request by adunndevster**

### CODE REVIEW

File: docs-authoring-pack/package.json [hunk 1]
```diff
@@ -26,7 +26,11 @@
...
     "docsmsft.docs-markdown",
     "docsmsft.docs-article-templates",
-    "streetsidesoftware.code-spell-checker"
+    "streetsidesoftware.code-spell-checker",
+    "docsmsft.docs-yaml"
   ],
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "vsce": "^1.58.0"
```
Comment (bharney-msft) [inline docs-authoring-pack/package.json]: Do we need this as a dependency? I though this was a global install

Comment (adunndevster) [reply docs-authoring-pack/package.json]: Yeah, removed :)

File: docs-markdown/package.json [hunk 2]
```diff
@@ -311,6 +311,7 @@
     "test-core": "node ./node_modules/vscode/bin/test"
   },
   "dependencies": {
+    "@types/jest": "^24.0.11",
```
Comment (bharney-msft) [inline docs-markdown/package.json]: This should be a dev dependency with the other types

File: docs-yaml/src/yaml-support/yaml-constant.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import * as path from 'path';
```
Comment (jamarw) [inline docs-yaml/src/yaml-support/yaml-constant.ts]: No big deal but we can change these to explicit imports if we follow the same style as the other extensions.  We can also enable TSLint to clean up some of the quotation/template string differences.

Comment (adunndevster) [reply docs-yaml/src/yaml-support/yaml-constant.ts]: Good idea!

File: docs-markdown/package.json (hunk 2 above)
Comment (adunndevster) [reply docs-markdown/package.json]: Good catch.

File: docs-yaml/src/yaml-support/yaml-constant.ts (hunk 3 above)
Comment (adunndevster) [reply docs-yaml/src/yaml-support/yaml-constant.ts]: @jamarw - Updated!

File: docs-markdown/package.json (hunk 2 above)
Comment (adunndevster) [reply docs-markdown/package.json]: @bharney-msft - updated!

**Pull request merged**